R/pipeline.R

```diff
@@ -210,15 +210,30 @@ execute_model_logic <- function(config, output_dir) {
     quantiles = unlist(config[["quantile_width"]])
   )
 
+  # All the top level metadata fields
+  metadata <- list(
+    job_id = config[["job_id"]],
+    task_id = config[["task_id"]],
+    data_path = config[["data"]][["path"]],
+    model = config[["model"]],
+    disease = config[["disease"]],
+    geo_value = config[["geo_value"]],
+    report_date = config[["report_date"]],
+    production_date = config[["production_date"]],
+    max_reference_date = config[["max_reference_date"]],
+    min_reference_date = config[["min_reference_date"]],
+    exclusions = config[["exclusions"]][["path"]],
```

Comment:
This is `None` by default -- https://github.com/CDCgov/cfa-config-generator/blob/main/utils/epinow2/constants.py#L13

Does this get written as "NULL" in R? Maybe worth casting to an empty string instead if so?

Reply:
A quick test shows that we get the following
```R
jsonlite::write_json(list(a_null_field=NULL), "test.json")
```
Gives
```json
{"a_null_field":{}}
```
I _think_ this is ok? What do you think @zsusswein?

Reply:
I'm getting some weird inconsistency writing/reading from the different formats 

### From R to R
```r
jsonlite::write_json(list(a_null_field=NULL), "test.json")
test <- jsonlite::read_json("test.json")
rlang::is_null(test$a_null_field)
#> FALSE
```

### From R to python

```py
import json

with open("test.json", "r") as f:
    d = json.load(f)
 d['a_null_field'] is None
# FALSE

d.items()
# dict_items([('a_null_field', {})])
```

### From python to R

```py
import json

test = {"a_null_field": None}
with open("test_py.json", "w") as f:
    json.dump(test, f)
```

And reading from R
```r
test_py <- jsonlite::read_json("test_py.json")
rlang::is_null(test_py$a_null_field)
#> TRUE
```

### From python to python


```py
import json

with open("test.json", "r") as f:
    d = json.load(f)
 d['a_null_field'] is None
# FALSE

 d['a_null_field'] is None
# TRUE
```

Headache!

Reply:
This is making me think that an empty string would be the best option